Clinical-Genomics/scout
# Check disease resource files before updating OMIM (and dropping old terms)

Features:
- Fixes loading of OMIM diagnoses in Scout demo instance (currently broken)
- Check that files needed for updating OMIM diagnosis contain valid data before dropping existing diagnoses in database

**How to test**:
1. Run `scout setup demo` command in master and in this branch and make sure that with this branch OMIM diagnoses are loaded in demo database.
2. Run `scout --demo update diseases --api-key some_key` and make sure that the command works normally 
3. Pre-download all files (`scout update everything --api-key some_key`) and update diseases using this command instead `scout --demo update diseases -f path/2/downloads/folder`. This command should work and update the OMIM terms as the command from point 2.
4. Bonus test: run: `scout --demo update diseases path/2/scout/demo/resources` and update scout demo with demo OMIM terms

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
